x-pack/plugins/cross_cluster_replication/public/app/sections/follower_index_edit/follower_index_edit.js

```diff
@@ -0,0 +1,232 @@
+          <EuiSpacer />
+          <EuiFlexGroup justifyContent="spaceAround">
+            <EuiFlexItem grow={false}>
+              <EuiButton
+                {...routing.getRouterLinkProps('/follower_indices')}
+                fill
+                iconType="plusInCircle"
+              >
+                <FormattedMessage
+                  id="xpack.crossClusterReplication.followerIndexEditForm.viewFollowerIndicesButtonLabel"
+                  defaultMessage="View follower indices"
+                />
+              </EuiButton>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </Fragment>
+      );
+    }
+
+    renderConfirmModal = () => {
+      const { followerIndexId, intl } = this.props;
+      const title = intl.formatMessage({
+        id: 'xpack.crossClusterReplication.followerIndexEditForm.confirmModal.title',
+        defaultMessage: 'Update follower index \'{id}\'',
+      }, { id: followerIndexId });
+
+      const { name, followerIndex } = this.inflightPayload;
+
+      return (
+        <EuiOverlayMask>
+          <EuiConfirmModal
+            title={title}
+            onCancel={this.closeConfirmModal}
+            onConfirm={() => this.props.saveFollowerIndex(name, followerIndex)}
+            cancelButtonText={
+              intl.formatMessage({
+                id: 'xpack.crossClusterReplication.followerIndexEditForm.confirmModal.cancelButtonText',
+                defaultMessage: 'Cancel',
+              })
+            }
+            buttonColor="danger"
+            confirmButtonText={
+              intl.formatMessage({
+                id: 'xpack.crossClusterReplication.followerIndexEditForm.confirmModal.confirmButtonText',
+                defaultMessage: 'Update',
+              })
+            }
+          >
+            <p>
+              <EuiIcon type="alert" color="danger" />{' '}
+              <FormattedMessage
+                id="xpack.crossClusterReplication.followerIndexEditForm.confirmModal.description"
+                defaultMessage="To update the follower index, it will first be paused and then resumed."
+              />
+            </p>
+          </EuiConfirmModal>
+        </EuiOverlayMask>
+      );
+    }
+
+    render() {
+      const {
+        clearApiError,
+        apiStatus,
+        apiError,
+        followerIndex,
+      } = this.props;
+
+      const { showConfirmModal } = this.state;
+
+      /* remove non-editable properties */
+      const { shards, ...rest } = followerIndex || {}; // eslint-disable-line no-unused-vars
+
+      return (
+        <EuiPage>
+          <EuiPageBody>
+            <EuiPageContent
+              horizontalPosition="center"
+              className="ccrPageContent"
+            >
+              <FollowerIndexPageTitle
+                title={(
+                  <FormattedMessage
+                    id="xpack.crossClusterReplication.followerIndex.editTitle"
+                    defaultMessage="Edit follower index"
```

Comment:
Do you think we should incorporate the name of the follower index, the same way we do in remote clusters? Or should we change remote clusters to just say "Edit remote cluster"?

![image](https://user-images.githubusercontent.com/1238659/51221779-b71ea500-18ef-11e9-97ed-b10258be61c6.png)

Reply:
remote clusters has been updated to just say `Edit remote cluster` 😄